pkg/cmd/admin/registry/registry.go

```diff
@@ -157,7 +158,12 @@ func NewCmdRegistry(f *clientcmd.Factory, parentName, name string, out io.Writer
 				Config: cfg,
 			}
 			kcmdutil.CheckErr(opts.Complete(f, cmd, out, args))
-			kcmdutil.CheckErr(opts.RunCmdRegistry())
+			err := opts.RunCmdRegistry()
+			if err != cmdutil.ErrExit {
```

Comment:
Generally we invert this:

```
if err == cmdutil.ErrExit {
  os.Exit(1)
}
kcmdutil.CheckErr(err)
```

That avoids the else block and simplifies the if so it is easier to read.